hack/run-functests.sh

```diff
@@ -22,14 +22,24 @@ export CLEAN_PERFORMANCE_PROFILE="false"
 # Latency tests env variables
 export LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-false}
 
-if [ "$FEATURES" == "" ]; then
-	echo "[ERROR]: No FEATURES provided"
-	exit 1
+echo "Running local tests"
+
+
+if [ "$DONT_FOCUS" == true ]; then
+	echo "DONT_FOCUS enabled, not focusing"
```

Comment:
meaning it will run all the tests?
let's be explicit about it
so ppl dont have to go hunt for ginkgo documentation on how the tool behave

Reply:
Yep, make sense.
Will change it

Reply:
Done